gradle/libs.versions.toml

```diff
@@ -51,6 +51,7 @@ guava = "33.4.7-jre"
 hadoop3 = "3.4.1"
 httpcomponents-httpclient5 = "5.4.3"
 hive2 = { strictly = "2.3.10"} # see rich version usage explanation above
+hive4 = { strictly = "4.0.1"}
```

Comment:
Is there a specific dependency on this hive version?  If this is strictly for big query, I think we need to scope it so that there isn't confusion with the hive module and potential dependencies there (e.g. `bigquery-hive4`)

Reply:
No There is not. We use constant looks like it is also available on hive2. I removed dependency.